From 2bc6e5ec75587a91bcdf8aef94ee09f15e9eb4f8 Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Tue, 25 Jun 2019 15:40:49 +0200
Subject: [PATCH] remove setup_requires

---
 setup.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/setup.py b/setup.py
index c066789..bc2dd51 100755
--- a/setup.py
+++ b/setup.py
@@ -121,7 +121,6 @@ def setup_package():
             url=about["__uri__"],
             license=about["__license__"],
             ext_modules=ext_modules,
-            setup_requires=["wheel>=0.32.0,<0.33.0"],
             classifiers=[
                 "Environment :: Console",
                 "Intended Audience :: Developers",
-- 
2.24.0.windows.2

